config/routes.rb

```diff
@@ -131,6 +131,8 @@
         patch 'blob/*path' => 'git#move_file', as: :git_move_file
         get 'freeze' => 'git#freeze_preview', as: :git_freeze_preview
         post 'freeze' => 'git#freeze', as: :git_freeze
+        patch '' => 'git#update', as: :git_update_version
+        #delete '' => 'git#destroy', as: :git_destroy_version
```

Comment:
is this intended to be commented out, and if so could just be removed ?

Reply:
Yeah it can probably be removed. I started looking into how removing versions works for non-git versioned things but wasn't sure if we are supporting that feature now that you can just hide versions.